packages/frontend/src/features/semanticViewer/components/RunSemanticQueryButton.tsx

```diff
@@ -100,20 +115,69 @@ export const RunSemanticQueryButton: FC = () => {
         sortBy,
     ]);
 
+    const handleSubmit = useCallback(
+        () =>
+            runSemanticViewerQuery({
+                ...allSelectedFieldsByKind,
+                sortBy,
```

Comment:
Missing limit here, when I change the limit I don't see it in the request payload

e.g. here I set the limit to 3 and I got all results

![image](https://github.com/user-attachments/assets/857a394d-8d17-40a2-bf37-8b3d18b2c8d1)


```suggestion
                sortBy,
                limit,
```

Reply:
Also on line 96 when we create the new transformer, should we be passing limit in the query?